src/assets/css/app.scss

```diff
@@ -0,0 +1,9 @@
+.app {
+    .center {
```

Comment:
Sass is a very exciting extension to CSS that allows you incorporate logic into your styling. I'm sure there have been times that you wished you could more easily format your CSS to show the relationship between parents and children, and Sass allows you to do that, among other exciting things. The only issue is that it needs to be transpiled, so it must be used in conjunction with Babel. That being said, all React applications require Babel, so there's no harm in including it in your app here! Great work!!